### DISCUSSION TITLE
Launching Discord Channel

### DISCUSSION THREAD

**Discussion by nutti** [nutti/fake-bpy-module#262] [Ideas]
Hi, @JonathanPlasse @Road-hog123 @Andrej730 

Thank you for your continuous contribution about this project.
Your contributions are very helpful to maintain this project.

Recently, the discussion is very active at many issues.
To make a faster and more interactive communication, I'm considering launching Discord channel.

How about this idea?

**Comment by JonathanPlasse** [comment]
I am all for it.
It would also be a good step to foster the spirit of high code quality for Blender add-ons.

**Comment by Road-hog123** [comment]
It would certainly be useful for asking questions about how the generation code works when trying to improve it. 👍 

Comments on discussions, issues, PRs and in the code are viewable without a GitHub account, so we should be careful not to hide useful information inside a Discord server.

**Reply by Andrej730** [reply]
Completely agree. Once I was trying to understand how the code generation works I was hesitant to ask any questions here but Discord server would help. And yes, typical problem with Discord is that it's good for communication but it's easy for information to get lost there.

**Comment by nutti** [comment]
@JonathanPlasse @Road-hog123 @Andrej730 

Thank you for your feedback!

I created the Discord channel for the discussion of fake-bpy-module.
https://discord.gg/dGU9et5S2d
I will edit the documentation later to be able to reach this channel from other potential users.

You can also ask the trivial question at this channel.
IMO, the discussion of issues / PRs (like bug fix or feature improvement) themselves should be performed at GitHub not to lose the discussion history.

If there are requests, please let me know.

**Comment by nutti** [comment]
Fortunately, the launched discord channel works very well.
I will close this discussion for now.
